docs/guides/1238-making-reusable-package.md

```diff
@@ -0,0 +1,188 @@
+---
+slug: /1238/making-reusable-package
+displayed_sidebar: "0.2"
+---
+
+# Making reusable packages
+
+Whilst splitting your plan into several files is a good idea, you will sometimes need to create standalone packages aiming to be reusable and shared. Let's explore how to do that.
+
+## Dagger project vs packages
+
+Understanding the difference between a project and a package is an important distinction, as it will help organizing your work.
+
+### Dagger project
+
+- Are use-case oriented: they solve a specific need through a set of actions
+- Aims to host a Dagger Plan
+- Contains a `cue.mod` folder
```

Comment:
That's a details but actually packages can also contains a `cue.mod` folder.
See [`dagger.io`](https://github.com/dagger/dagger/tree/main/pkg/dagger.io) for instance or `universe.dagger.io`

Reply:
it becomes a module, to reference them locally